### ISSUE TITLE
KVM live migration can fail with different KVM versions between source/target

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#371]
Originally reported of Google Code with ID 315.

```
KVM can do live migration across versions, but not across all versions; users report that migrations can fail silently in some cases.

We should have some checks or at least warnings for the user.
```
Originally added on 2012-11-19 08:19:13 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Commit e6ba332 addresses this. Is it enough, in your opinion?
```
Originally added on 2013-01-11 08:57:35 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-01-11 17:36:08 +0000 UTC.

Added Labels: Hypervisor-KVM
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking as fixed as no further comment was added, and the option to specify the machine type for cross-migrations was added.
```
Originally added on 2013-02-26 16:03:54 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
